custom/panel_templates/Default/core/debugging_and_maintenance.tpl

```diff
@@ -134,7 +134,7 @@ $('#debug_link').click(() => {
             toastr.options.progressBar = true;
             toastr.options.closeButton = true;
             toastr.options.positionClass = 'toast-bottom-left';
-            toastr.info('{$TOASTR_COPIED}'.replaceAll({literal}'{x}'{/literal}, url));
+            toastr.info('{$TOASTR_COPIED}'.replaceAll({literal}'{{link}}'{/literal}, url));
```

Comment:
I think, in this case, it does need to be `{x}` (see line 171 of modules/Core/pages/panel/debugging_and_maintenance.php)

Reply:
Good catch